File: server/src/app/functions/commands/tutorial.ts
import { messages } from '@server/config/messages';
import config from '@server/config/env';

export function tutorial(wamName: string, callerId: string) {
  return {
    result: {
      type: 'wam',
      attributes: {
        appId: config.appId,
        name: wamName,
        wamArgs: { message: messages.tutorialMsg, managerId: callerId },
      },
    },
  };
}

File: server-fast/app/functions/commands/tutorial.py
from ...config.messages import messages
from ...config.env import config

def tutorial(wam_name: str, caller_id: str):
    return {
        "result": {
            "type": "wam",
            "attributes": {
                "appId": config["APP_ID"],
                "name": wam_name,
                "wamArgs": {"message": messages["tutorial_msg"], "managerId": caller_id}
            }
        }
    }